done CustomerController, CustomerService and CustomerRepository

FertGL opened the pull request:
done CustomerController, CustomerService and CustomerRepository

FertGL committed:
done CustomerController, CustomerService and CustomerRepository

Ceasarshin commented on src/main/java/Conroller/CustomerController.java:
```diff
@@ -0,0 +1,21 @@
+package Conroller;
```
Исправь Controller

Ceasarshin commented on src/main/java/Conroller/CustomerController.java:
```diff
@@ -0,0 +1,21 @@
+
+import Exception.CustomerException.CustomerNotFoundException;
+import Model.Customer.CustomerType;
+import Reporitory.CustomerRepository;
```
Repository

FertGL committed:
feather 45/46
added javadoc, done CustomerController
added some checks to getAllCustomers()

Ldv236 commented on src/main/java/Conroller/CustomerController.java:
```diff
@@ -0,0 +1,21 @@
+        customerService.getAllCustomers();
+        customerService.getCustomerById(2);
+
+        //Проверка работы всех методов
```
потом уберешь же эти вызовы?

Ldv236 commented on src/main/java/Model/Customer/CustomerType.java:
```diff
@@ -0,0 +1,7 @@
+package Model.Customer;
+
+public enum CustomerType {
+    NEW,
```
👍

Ldv236 commented on src/main/java/Exception/CustomerException/CustomerNotFoundException.java:
```diff
@@ -0,0 +1,12 @@
+    }
+
+    @Override
+    public String getMessage() {
```
не, так не надо, ни разу не видел чтоб этот метод переопределяли, он и так работает как надо

Ldv236 replied:
и почему два эксепшена, не пойму

Ldv236 commented on src/main/java/Repository/CustomerRepository.java:
```diff
@@ -0,0 +1,49 @@
+    public Customer add(Customer customer) {
+        customer.setId(++customerCount);
+        customerLinkedHashMap.put(customer.getId(), customer);
+        System.out.println("Покупатель добавлен, присвоен ID - " + customer.getId());
```
ок, но потом поменяем на запись логов, а не просто сообщения в консоль

Ldv236 commented on src/main/java/Repository/CustomerRepository.java:
```diff
@@ -0,0 +1,49 @@
+     *
+     * @return LinkedHashMap
+     */
+    public Map<Integer, Customer> allCustomers() {
```
в начале названия метода глагол - например, гет

FertGL committed:
feather 45/46
fix CustomerNotFoundException (delete getMessage())
rename AllCustomers() to getAll
